data/sample_generated_seed/sample_operations.csv

```diff
@@ -0,0 +1,100 @@
+GET,11
+PUT,375,975026

```

Comment:
Lets decrease the value size to <= 100000 as previously discussed. Values that are too large can risk decryption throwing an error which is sometimes seen in enclave.

Reply:
I'll make this a separate PR